src/main/resources/db/migration/V2024_03_15_1445__CIV-13077_defendant_orgorltdcompany_admit_pay_by_date_scenario.sql

```diff
@@ -0,0 +1,28 @@
+/**
+ * Add scenario
+ */
+INSERT INTO dbs.scenario (name, notifications_to_delete, notifications_to_create)
+VALUES ('Scenario.AAA7.DefResponse.OrgOrLtdCompany.FullOrPartAdmit.PayBySetDate.Claimant',
+        '{"Notice.AAA7.DefResponse.ResponseTimeElapsed.Claimant", "Notice.AAA7.ClaimIssue.Response.Await",
+         "Notice.AAA7.ClaimIssue.HWF.PhonePayment", "Notice.AAA7.DefResponse.MoretimeRequested.Claimant", "Notice.AAA7.ClaimIssue.HWF.FullRemission"}',
+        '{"Notice.AAA7.DefResponse.OrgOrLtdCompany.FullOrPartAdmit.PayBySetDate.Claimant" : ["defendantAdmittedAmount", "respondent1AdmittedAmountPaymentDeadlineEn", "respondent1AdmittedAmountPaymentDeadlineCy"]}');
+
+/**
+ * Add notification template
+ */
+INSERT INTO dbs.dashboard_notifications_templates (template_name, title_En, title_Cy, description_En, description_Cy
+                                                  ,notification_role)
+VALUES ('Notice.AAA7.DefResponse.OrgOrLtdCompany.FullOrPartAdmit.PayBySetDate.Claimant', 'Response to the claim', 'Response to the claim',
+        '<p class="govuk-body">The defendant has offered to pay ${defendantAdmittedAmount} by ${respondent1AdmittedAmountPaymentDeadlineEn}.</p><p class="govuk-body">The defendant needs to send you their financial details.<br><a href="{CLAIMANT_RESPONSE_TASK_LIST}" class="govuk-link">View and response</a>.</p>',
```

Comment:
"...<.br>" is necessary here? we can standardize using "..<.p>"

Reply:
removed this entry